unittests/resources/checks_unlisted/externalvars.py

```diff
@@ -8,7 +8,7 @@ class external_x(rfm.RunOnlyRegressionTest):
     valid_systems = ['*']
     valid_prog_environs = ['*']
     foo = variable(int, value=1)
-    ham = variable(bool, value=False)
+    ham = variable(Bool, value=False)
```

Comment:
How did this work?

```suggestion
    ham = variable(typ.Bool, value=False)
```

Reply:
I did not work actually, I need to add one more assertion to make sure that the tests are actually loaded without any problem with the declaration of the variables.